src/components/molecules/Install/Install.test.tsx

```diff
@@ -1,148 +1,59 @@
-import { describe, it, expect, vi } from 'vitest';
+import { describe, it, expect, vi, beforeEach } from 'vitest';
 import { render, screen, fireEvent } from '@testing-library/react';
-import { InstallCard, InstallFooter, InstallHeader, InstallNavigation } from './Install';
-import * as Config from '@/config';
+import { InstallCard, InstallHeader, InstallNavigation } from './Install';
 import * as App from '@/app';
 
-// Mock Next.js router
-const mockPush = vi.fn();
-vi.mock('next/navigation', () => ({
-  useRouter: () => ({
-    push: mockPush,
-  }),
-}));
+// Helper function to normalize Radix UI IDs for snapshot testing
+const normalizeRadixIds = (html: string): string => {
```

Comment:
This already exists in https://github.com/pubky/franky/blob/6e02be58bb8702b55574fae1c118996817dc6da6/src/libs/utils/utils.ts#L73